### ISSUE TITLE
Autoresize does not work when container is grid

### ISSUE THREAD

**Issue by radusuciu** [ecomfe/vue-echarts#675]
### Confirmation

- [X] I can confirm this problem is not reproducible with ECharts itself.

### How are you introducing Vue-ECharts into your project?

ES Module imports

### Versions

```sh
+-- @tanstack/vue-table@8.5.22
| `-- vue@3.2.45 deduped
+-- @vitejs/plugin-vue@3.2.0
| `-- vue@3.2.45 deduped
+-- @vueuse/core@9.5.0
| +-- @vueuse/shared@9.5.0
| | `-- vue-demi@0.13.11
| |   `-- vue@3.2.45 deduped
| `-- vue-demi@0.13.8
|   `-- vue@3.2.45 deduped
+-- echarts@5.4.0
+-- floating-vue@2.0.0-beta.20
| +-- vue-resize@2.0.0-alpha.1
| | `-- vue@3.2.45 deduped
| `-- vue@3.2.45 deduped
+-- pinia@2.0.27
| +-- vue-demi@0.13.6
| | `-- vue@3.2.45 deduped
| `-- vue@3.2.45 deduped
+-- sortablejs-vue3@1.2.3
| `-- vue@3.2.45 deduped
+-- vue-echarts@6.2.3
| +-- echarts@5.4.0 deduped
| +-- vue-demi@0.13.5
| | `-- vue@3.2.45 deduped
| `-- vue@3.2.45 deduped
+-- vue-router@4.1.3
| `-- vue@3.2.45 deduped
`-- vue@3.2.45
  `-- @vue/server-renderer@3.2.45
    `-- vue@3.2.45 deduped
```

### Details

The autoresize does not work when the containing element has `display:grid` set and the window is resized to be smaller. It only kind of works when the size increases.

![autoresize](https://user-images.githubusercontent.com/1108600/205152912-fdb1a3dd-eeb0-438d-af43-8562317af900.gif)

### Reproduction

https://stackblitz.com/edit/vue-echarts-vue-3-dfksyi?file=src%2FApp.vue

**Comment by Justineo** [comment]
Try add this in your CSS to workaround the issue:

```css
x-vue-echarts {
  overflow: hidden;
}
```

**Comment by radusuciu** [comment]
This definitely works in the example I posted! Thank you.

For my actual application I ended up using: https://vueuse.org/core/useResizeObserver/

**Comment by jhxxs** [comment]
> This definitely works in the example I posted! Thank you.
> 
> For my actual application I ended up using: https://vueuse.org/core/useResizeObserver/
Thanks, this really helps.
![image](https://user-images.githubusercontent.com/7448223/207221531-f8f42742-5c6c-4680-af23-e8c49cd603c3.png)

**Comment by Justineo** [comment]
If you are using an external resize observer library, you may want to disable the built-in `autoresize` as you'll end up calling `chart.resize()` twice in many cases.

**Comment by Justineo** [comment]
There seems to be another fix for this:

```css
x-vue-echarts {
  min-width: 0;
}
```

@radusuciu @jhxxs Can you try this fix in your projects? The external resize observer seems to be unnecessary.

**Comment by Justineo** [comment]
Fixed in v6.3.3.
